hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -1477,6 +1477,7 @@ private void disableCache() {
       // If persistent ioengine and a path, we will serialize out the backingMap.
       this.backingMap.clear();
       this.fullyCachedFiles.clear();
+      this.regionCachedSizeMap.clear();
```

Comment:
I think we also need to do this [here](https://github.com/apache/hbase/pull/5376/files#diff-b75abcdb76c582e16144df3a9bf2ddbc8fd0814c06190c33503a2c1cb365273cL342)

Reply:
Done. Added code to clear the regionCachedSizeMap